lib/data_cycle/constant_update.rb

```diff
@@ -113,6 +113,7 @@ def remove_needs_update_flags
   end
 
   def conflicting_updates_running?
+    return true if update_running?(:short)
```

Comment:
Added this here in order to prevent conflicts. I set the short one as the first one as is expected to be the shortest one.

Reply:
If we can't run the short update concurrently with other updates, then splitting off a separate process probably won't help much; we'll still need to stop short updates for ~5 hours every time constant update runs, and longer for daily update.

I think instead we should fix any conflicts such that we can be running short updates on a continual basis, without worrying about what other updates are running.